src/tools/illink/src/ILLink.RoslynAnalyzer/DataFlow/LocalDataFlowVisitor.cs

```diff
@@ -85,7 +85,10 @@ public void Transfer (BlockProxy block, LocalDataFlowState<TValue, TValueLattice
 
 			// Use the branch value operation as the key for the warning store and the location of the warning.
 			// We don't want the return operation because this might have multiple possible return values in general.
-			HandleReturnValue (branchValue, branchValueOperation);
+			Debug.Assert (OwningSymbol is IMethodSymbol);
```

Comment:
This feels weird - where does it go through for attributes? Through the `if (branchValueOperation == null)`?